.eslintrc

```diff
@@ -121,11 +121,16 @@
     "plugin:react/jsx-runtime",
     "plugin:react-hooks/recommended",
     "plugin:import/errors",
-    "plugin:import/warnings"
+    "plugin:import/warnings",
```

Comment:
Why? This was added in this PR because we're importing `immer` for the first time, and it importing it created the lint error: `error  Unable to resolve path to module 'immer'  import/no-unresolved`. To be fair I don't understand how this works, but Alex P suggested it [here](https://metaboat.slack.com/archives/C505ZNNH4/p1692635162528689?thread_ts=1692631233.986049&cid=C505ZNNH4)

Reply:
now it's clear, thanks for explanation